### DISCUSSION TITLE
builtin_tasks.remove_old_jobs error

### DISCUSSION THREAD

**Discussion by vedanthshetty-739** [procrastinate-org/procrastinate#1073] [Q&A]
Hi Team,
I'm using Procrastinate in one of my project.
earlier i was using version 0.29.1 now moved to 2.30.0.
Now I'm getting error while calling this built in `remove_old_jobs` to clean Procrastinate record., 
while debugging I found it's failing in sql query.
error: `WHERE job.status = ANY(ARRAY['succeeded','failed'])` 

Could you please let us know on this issue once.

**Comment by ewjoachim** [comment]
This is an issue, more than a discussion, let's continue [here](https://github.com/procrastinate-org/procrastinate/issues/1075).
